AppInspector.CLI/Writers/VerifyRulesJsonWriter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.ApplicationInspector.Commands;
+using Newtonsoft.Json;
+
+namespace Microsoft.ApplicationInspector.CLI.Writers
+{
+    class VerifyRulesJsonWriter : CommandResultsWriter
+    {
+        public override void WriteResults(Result result, CLICommandOptions commandOptions, bool autoClose = true)
+        {
+            WriteOnce.Result("Result");
+
+            JsonSerializer jsonSerializer = new JsonSerializer();
+            jsonSerializer.Formatting = Formatting.Indented;
+            jsonSerializer.Serialize(TextWriter, (VerifyRulesResult)result);
+
+            WriteOnce.NewLine();
+
+            if (autoClose)
+                FlushAndClose();
+        }
+
+        public override void FlushAndClose()
```

Comment:
You seem to do the same thing in all of them, do you need to override this? Could it just be part of the base class?

Reply:
For most json writers other than analyze yes.  I'll take care of to reduce.

Reply:
fixed